.travis.yml

```diff
@@ -1,9 +1,21 @@
-language: node_js
-node_js: 12
+language: rust
+rust:
+  - stable
+  - nightly
+
 env:
   - NODE_ENV=ci
   - NODE_ENV=ci-staging
 cache: yarn
+
+before_install:
+  - rustup target add wasm32-unknown-unknown
+  - nvm install 10.16
+  - nvm use 10.16
+
+install: 
+  - npm install
```

Comment:
Should use `yarn` given we use `yarn` everywhere else (including `cache: yarn`). yarn can be installed in `before_install` using `npm install -g yarn`

Reply:
done